### PR TITLE
Countdown timer

### PR DESCRIPTION
**Pull request by ybatsiun**

### CODE REVIEW

File: client_react/src/components/gamePage/CountdownTimer.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+function getTimeRemaining(deadline) {
+    const localOffset = new Date().getTimezoneOffset();
+    const total = moment(deadline)
+        .subtract(localOffset, 'minutes')
+        .diff(moment(), 'milliseconds');
+    const seconds = Math.floor((total / 1000) % 60);
+    const minutes = Math.floor((total / 1000 / 60) % 60);
+    const hours = Math.floor((total / (1000 * 60 * 60)));
+    ///const days = Math.floor(total / (1000 * 60 * 60 * 24));
```
Comment (JackKarichkovskiy) [inline client_react/src/components/gamePage/CountdownTimer.js:32]: Why days are commented?

File: server/src/main/resources/application.yaml [hunk 2]
```diff
@@ -59,7 +59,7 @@ cors:
 
 voting:
   # 5 min
-  timeToVoteInSeconds: 300
+  timeToVoteInSeconds: 3000
```
Comment (JackKarichkovskiy) [inline server/src/main/resources/application.yaml:62]: Can you return to 5 min please? Or it was done by intent?

**Review by JackKarichkovskiy — CHANGES_REQUESTED** [review]

File: client_react/src/components/gamePage/CountdownTimer.js (hunk 1 above)
Comment (ybatsiun) [reply client_react/src/components/gamePage/CountdownTimer.js:32]: I decided to follow such pattern:  hours:minutes:seconds
So if there are more than 1 day left for voting: timer will show (for example) 35 hours 2 minutes 56 seconds

File: server/src/main/resources/application.yaml (hunk 2 above)
Comment (ybatsiun) [reply server/src/main/resources/application.yaml:62]: done

File: client_react/src/components/gamePage/CountdownTimer.js (hunk 1 above)
Comment (JackKarichkovskiy) [reply client_react/src/components/gamePage/CountdownTimer.js:32]: Ok then, looks good

**Review by JackKarichkovskiy — APPROVED** [review]

**Pull request merged**